### ISSUE TITLE
Kotlin Dataframe introduces a dependancy on org.slf4j.slf4j-simple

### ISSUE THREAD

**Issue by jebbench** [Kotlin/dataframe#940]
The Dataframe library introduces a transitive dependancy on `org.slf4j.slf4j-simple` when included in our project, this results in errors (and a chance to choose the wrong logging framework) at runtime:
```
SLF4J(W): Class path contains multiple SLF4J providers.
2024-10-31T15:09:04.359347693Z SLF4J(W): Found provider [org.apache.logging.slf4j.SLF4JServiceProvider@c4d2c44]
2024-10-31T15:09:04.359357157Z SLF4J(W): Found provider [org.slf4j.simple.SimpleServiceProvider@13866329]
2024-10-31T15:09:04.359361564Z SLF4J(W): See https://www.slf4j.org/codes.html#multiple_bindings for an explanation.
```

We have worked around this by excluding the dependancy in our Gradle build file.
```
    implementation("org.jetbrains.kotlinx:dataframe:$dataframe_version") {
        exclude group: 'org.slf4j', module: 'slf4j-simple'
    }
    implementation("org.jetbrains.kotlinx:dataframe-excel:$dataframe_version") {
        exclude group: 'org.slf4j', module: 'slf4j-simple'
    }
```

Using dataframe version 0.14.1.

I believe the fix is to depend on `slf4j-api` in libraries.

**Comment by Jolanrensen** [comment]
Already handled by https://github.com/Kotlin/dataframe/pull/934

**Comment by Jolanrensen** [comment]
It's released along the hot-fix version [0.14.2](https://github.com/Kotlin/dataframe/releases/tag/v0.14.2).
